src/annotator/adder.tsx

```diff
@@ -105,34 +109,15 @@ export class Adder {
       left: 0,
     });
 
-    this._view = /** @type {Window} */ (element.ownerDocument.defaultView);
-
-    this._width = () => {
-      const firstChild = /** @type {Element} */ (this._shadowRoot.firstChild);
-      return firstChild.getBoundingClientRect().width;
-    };
-
-    this._height = () => {
-      const firstChild = /** @type {Element} */ (this._shadowRoot.firstChild);
-      return firstChild.getBoundingClientRect().height;
-    };
```

Comment:
I haven't seen a clear reason to have `_width` and `_height` as arrow function props instead of regular methods, so I have migrated them to be methods (a bit below).

Let me know if it is preferred to keep these as props.

Reply:
As long as tests pass (and tests are good tests, assuming), do whatever you like in this respect!